renovate.json5

```diff
@@ -17,18 +17,6 @@
   //   old branch point. GH auto-merge doesn't have that property so it's a bit more likely that
   //   a deps update could break the build if there are recent related changes.
   platformAutomerge: false,
-  // We've run into some weird problems where Renovate will notice that a
```

Comment:
Did you change something to fix this issue? The problem we were seeing was that sometimes Renovate/npm would end up saying "oh hey, foo@1.2.3 is in both top-level and docs-level node_modules, let's dedup things so that isn't installed twice".  But Netlify only runs `npm i` in the docs subdirectory so the package wouldn't get installed at all for Netlify and the build would fail.

Even if you haven't fixed anything I'm fine with you making this change, since we can always revert it if the problems happen again (they're really obvious, Netlify builds just all start failing).

Reply:
Ahh I did not! Thank you for catching that, I had assumed that just updating a previous configuration would be sufficient. I'll add this part back in.